### PR TITLE
document r2dbc-pool usage

### LINKED ISSUE #158: Document how to use r2dbc-pool and configure connection keep-alive
Follow-up to #141.

### PR DESCRIPTION
**Pull request by dmitry-s**
fixes #158

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,30 @@ The easiest way to start using the driver is to add the driver dependency throug
 </dependency>
 ```
 
+**Repositories**
+
+```xml
```
Comment (meltsufin) [inline README.md]: Can you mark the above one as `xml` as well for consistency?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,30 @@ The easiest way to start using the driver is to add the driver dependency throug
 </dependency>
 ```
 
+**Repositories**
```
Comment (meltsufin) [inline README.md]: Comment why these repositories are needed?
I thought you would just add them in the Connection Pool section.

Also, equivalent for Gradle?

File: README.md [hunk 3]
```diff
@@ -396,3 +420,45 @@ Flux.from(connection.createBatch()
     .execute())
     .flatMap(r -> r.getRowsUpdated());
 ```
+
+## Using Connection Pool
+For connection pooling, [r2dbc pool](https://github.com/r2dbc/r2dbc-pool) can be used.
```
Comment (meltsufin) [inline README.md]: Maybe a comment on why you might want to use the pool and that it takes care of validating that connections are not stale?

File: pom.xml [hunk 4]
```diff
@@ -304,6 +310,22 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```
Comment (meltsufin) [inline pom.xml]: comment that this is for the pool?

Comment (dmitry-s) [reply pom.xml]: done

File: README.md (hunk 3 above)
Comment (dmitry-s) [reply README.md]: done

**Pull request merged**